leafmap/ai.py

```
@@ -0,0 +1,774 @@
+    parser.add_argument("--no-execute", action="store_true", help="只生成代码不执行")
+    args = parser.parse_args()
+
+    natural_map(
+        args.description,
+        output=args.output,
+        execute=not args.no_execute,
+    )
+
+
+def demo():
+    """启动 Gradio Web Demo：在浏览器中通过自然语言生成地图。
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)